config/community-resources.json

```
@@ -98,5 +98,15 @@
       "name": "Maximising Your Job Prospects in Tech with BioDrop",
       "url": "https://loftwah.medium.com/maximising-your-job-prospects-in-tech-with-BioDrop-2fea408301cc"
     }
+  },
+  {
+    "name": "Nafisa Faruna",
```

The name here should be the creator of the resource

```suggestion
    "name": "Kim McMahon",
```